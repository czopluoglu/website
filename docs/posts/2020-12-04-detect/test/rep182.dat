01000001111010000001
11111111011000000110
01010001110111101111
01001101001111110111
10111101111111111111
11001010100100100011
11111101001100010111
10111010110000100010
10111101111100111111
11101011111111101111
10001111101111111110
10001101111100101111
10011111000010001101
00000001011100000111
11010010111110100010
01010011001001000111
01100011110001110110
01101000110001101010
00001101111110010111
10010001010011100101
11011010000111101111
00011011101000100011
10001000011001100011
01011111110100000010
10000101001111000111
11100101011000110100
11110101111011111110
11001001111111111001
10001011001000001010
11011111011000001010
11110111111110001101
10001101111011001110
00011011010011100010
01100001110010110111
10011101110000000111
11000000111001111011
00001101111110010100
01100100011001000110
00111000100011011011
01010100100001110010
10010111111010100110
11111111111110110110
01011001011011100001
00001001100000001010
11010101110010000010
11000000001010110011
11111111111011111011
10000001100111110101
11101000010011000110
11111010001011000100
01101001111010101000
00110100100011111001
01001101111111011111
01111000111000000010
10000000001000011010
10110011111010110111
10000000000000000110
00001001111010101101
01110000001111111111
11000000001000000010
00001100000101101111
10000111101000111110
11111000110010101111
11000101010111010111
01111110111100011010
11001001110110100111
10001101001001111111
00010101111000000000
00000011000111011101
00100001110111000110
00010011010000100111
01101011001110100011
10010101000011100110
10010100111011101100
00001001010000100111
10000101011010111111
00111101001111111001
10101110011101000110
01100101111110000000
01001101000111100101
11111111111101111111
00011011101111010001
00101100101101111111
11111010011000000010
01101001011010010111
00000010001111111111
11001100001010111000
00001001011010101111
11000101101110111011
10000001101100100000
01000001111011011111
11100001000000001011
00001101011011001101
10111101011110000111
11011101011000010111
10111100110010101101
10011001101111101111
01010000100000101000
10100101101011101111
10110101010111111111
00100001010000001010
11111110111010110111
11110000100100010011
11000001010010100010
00101111101001101101
11101010111011100011
00010111111000000001
11011001101110011100
10010000000001100010
01100100111010010110
10010101111000000011
01001100101111100111
01001000000000110110
00000101000000000011
11000100011010100011
00011111000001110011
00000000001011011010
11001001001010111010
10111111101111110011
00000100100001100011
10111111110110110111
01000001101101000011
11000111101010111010
01111111101010110100
10100000111010000110
11110011101010100001
10101111111010100110
01010000011010101110
00001001001001011110
10101001111110101111
10011111111011111111
01010010111000000000
11010000110101100110
00001101010010000100
10000100001010111101
11011100011101011011
10110111110111111111
10010111110011110011
01010111011100110011
11010101111110001111
11111011111110101111
01000001001010100111
10001000110011000010
10001000001001101001
11000111101001100110
10000110101000110110
10111111101011101111
10000000111111001011
00001111000101101111
00101111111111111110
11001110000010100010
11010101110011100011
00010000001010101111
11001001001001000001
00101101010010110110
11001101111111100110
10001001101110110100
11111111111111011101
11010111001111011110
10111110111010110111
10011011111110101111
00001001001011001100
00011010001110010110
11100100010000000010
00000111111110111111
10010111111010110011
01001010011010000010
01001010001001000001
11111000111111111111
10001001001111101100
10111011111000111100
10011111111101100111
10011101011000111111
11111011111001110010
10000010111110111100
01101001011011110110
10100000110000110100
00010101101100001110
01011111111101001111
01101101111010010000
01010000111010101111
01010101111110011101
00010011111111111111
11110111111111111101
00000100111100001010
00011101101011111111
10110101001010010111
11111001111100111011
00010001011100001000
01100110100110001010
10101100011010111001
01011011011110101010
11001010011010101001
10000110011010101100
01111111011000100000
00110101101000101111
10010001100010110000
00000000100100101101
10010111101110101111
00011100001100000000
01110011011111110111
11100111111111110111
01010111111011111001
00001000010000110110
01000100101100011010
10001101001110111111
10110000001110111110
10000111111000010110
11000010011111111110
00001000001111111111
01010110110010011001
11111000001101111011
11011101011101000011
11111111111000100101
00111101101001000111
01111111111010011111
01000101001011111010
01111101101111011101
10100111001001011100
10101101001100111111
10011001111111101111
11111000011000010110
00011100111110011111
01110101111000101010
11000001111111000110
11001001010011010000
10000001011110101111
11110111100001100111
11010101011010100010
10010100101010101011
01011010010010010011
10001001101010000011
10101010101010001100
11001011010100001111
11111011110010101110
10101111101111111110
10101111110111111111
00010011011111011111
11100101111000100110
10010011011010000101
10000000100000001011
01000100001011010110
11011110111010000111
00010111101111101110
01001001010010000100
11111100010110111111
00000001000010000011
00000001010011100111
01110110011100101101
11001010100110101100
