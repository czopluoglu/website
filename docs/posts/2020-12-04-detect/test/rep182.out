-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep182.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.40
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8942
                   IDN index value:   0.7263
                           Ratio r:   0.6762

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    2    3    4    2    4    2    2
    4    5    5    5    5    5    5    5    1    5

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1   19

   -----------CLUSTER  2-------------

    2    3    4    7    9   10

   -----------CLUSTER  3-------------

    5

   -----------CLUSTER  4-------------

    6    8   11

   -----------CLUSTER  5-------------

   12   13   14   15   16   17   18   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+-+-+++-+-+---+----
   +d+----+-+--+-++--++
   -+d+++-+------------
   +-+d++++++------+-+-
   --++d+++-++--------+
   +-+++d++++++---+----
   +--+++d+-++-+--+-+--
   +++++++d--+--+----+-
   ---+-+--d--++-+----+
   ++-++++--d++----+-+-
   ----++++-+d++++---++
   +----+--+++d+---++--
   -+----+-+-++d++-++++
   -------+--+-+d++++++
   -+------+-+-++d+++++
   ++---++------++d++++
   ---+-----+-+++++d++-
   ------+----++++++d++
   -+-+---+-++-++++++d+
   -+--+---+-+-++++-++d
 
 No cross validation for this DETECT run
